apps/ledger-live-desktop/src/renderer/components/CustomImage/NFTGallerySelector.tsx

```diff
@@ -28,14 +30,35 @@ type Props = {
 };
 
 const NFTGallerySelector = ({ handlePickNft, selectedNftId }: Props) => {
+  const nftsFromSimplehashFeature = useFeature("nftsFromSimplehash");
+  const threshold = nftsFromSimplehashFeature?.params?.threshold;
+  const accounts = useSelector(accountsSelector);
   const nftsOrdered = useSelector(orderedVisibleNftsSelector, isEqual);
+
+  const addresses = useMemo(
+    () =>
+      [
+        ...new Set(
+          accounts.map(account => account.freshAddress).filter(addr => addr.startsWith("0x")),
+        ),
+      ].join(","),
+    [accounts],
+  );
+
+  const { nfts, fetchNextPage, hasNextPage } = useNftGalleryFilter({
+    nftsOwned: nftsOrdered || [],
+    addresses: addresses,
+    chains: ["ethereum", "polygon"],
+    threshold: isThresholdValid(threshold) ? Number(threshold) : 75,
+  });
+
   const { t } = useTranslation();
 
   const [displayedCount, setDisplayedCount] = useState(10);
 
   const content = useMemo(
     () =>
-      nftsOrdered.slice(0, displayedCount).map((nft, index) => {
+      nfts.slice(0, displayedCount).map((nft, index) => {
```

Comment:
We have to think if for whatever reason we turn off FF

nftsFromSimplehashFeature?.enabled ? nfts : nftsOrdered